### PR TITLE
Update changelog for MSAL 481

### PR DESCRIPTION
**Pull request by trwalke**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,21 @@
+4.8.0.
+============
+Bug fix:
+**Cannot acquire token in UWP app on HoloLens via a unity plugin as json serialization fails** MSAL now uses Newtonsoft.Json for serialization. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1468)
```
Comment (jmprieur) [inline changelog.txt]: I would not write it uses Newtonsoft.JSon (as it's really using the internal namespace). Just write that the serialization is now fixed.

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+
+**AAD Security question registration page unresponsive in Android embedded webview** MSAL now properly handles the andoid activity when usin the embedded webview. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1532)
```
Comment (jmprieur) [inline changelog.txt]: usin => using

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+
+**"offline_access" scope causes token cache misses** MSAL now Proberly filters the cache during silent authentication. [Issues](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1548)
```
Comment (jmprieur) [inline changelog.txt]: Proberly => properly

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+
+**Improved invalid client error message** MSAl now has a better error message when an invalid client error is sent back from AAD. [Issues](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1576)
```
Comment (jmprieur) [inline changelog.txt]: MSAl => MSAL.NET

File: changelog.txt [hunk 5]
```diff
@@ -1,3 +1,21 @@
...
+
+New Features:
+**MSAL.NET now supports Proof of Possession** The PublicClientApplication on every target has support for this. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1488)
+
+**Token cache serialization for Mac** MSAl is now able to serialize and deserialize the token cahce auring authentication on MAC OS. [Issues](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1487)
```
Comment (jmprieur) [inline changelog.txt]: MSAl => MSAL
cahce => cache
auring => during

File: changelog.txt [hunk 6]
```diff
@@ -1,3 +1,21 @@
...
+
+**MSAL now uses "mobile" configuration for iOS Xamarin embedded webview** MSAl now properly uses the MKWebview on iPad when using the embedded webview. [Issues](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1552)
```
Comment (jmprieur) [inline changelog.txt]: MSAl => MSAL

Comment (jmprieur) [inline changelog.txt]: MKWebview => WKWebview

File: changelog.txt [hunk 7]
```diff
@@ -1,3 +1,21 @@
...
+**Cannot acquire token in UWP app on HoloLens via a unity plugin as json serialization fails** Serialization now works properly in MSAL.NET. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1468)
+
+**AAD Security question registration page unresponsive in Android embedded webview** MSAL.NET now properly handles the andoid activity when using the embedded webview. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1532)
+
+**"offline_access" scope causes token cache misses** MSAL.NET now properly filters the cache during silent authentication. [Issues](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1548)
+
+**Improved invalid client error message** MSAL.NET now has a better error message when an invalid client error is sent back from AAD. [Issues](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1576)
+
+New Features:
+**MSAL.NET now supports Proof of Possession** The PublicClientApplication on every target has support for this. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1488)
```
Comment (bgavrilMS) [inline changelog.txt]: Only on net45.

File: changelog.txt [hunk 8]
```diff
@@ -1,3 +1,21 @@
...
+
+**Token cache serialization for Mac** MSAL.NET is now able to serialize and deserialize the token cache during authentication on MAC OS. [Issues](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1487)
```
Comment (bgavrilMS) [inline changelog.txt]: This is not correct. The work done there was to validate the MSAL extension which adds a token cache serialization mechanism. No work has been done on MSAL.

File: changelog.txt [hunk 9]
```diff
@@ -1,3 +1,21 @@
+4.8.1.
```
Comment (bgavrilMS) [inline changelog.txt]: Are we not looking at fixing https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1487 before releasing?

Comment (jennyf19) [reply changelog.txt]: i'm looking at it now

File: changelog.txt (hunk 8 above)
Comment (trwalke) [reply changelog.txt]: oh ok, ill fix

**Pull request merged**